TCA Counter App/EditCount/CountFeature.swift

```diff
@@ -8,10 +8,12 @@
 import Foundation
 import ComposableArchitecture
 
-struct CounterState: Equatable {
+struct CounterState: Equatable, Identifiable {
+    var id = UUID()
     static func == (lhs: CounterState, rhs: CounterState) -> Bool {
         return lhs.count == rhs.count
     }
```

Comment:
I think in general we want to rely on the default equatable conformance's implementation. In this case we are leaving out the alert from this check thus states where alert == nil and alert != nil will be treated as equal.

Reply:
cool~ Thanks.